src/analyses/libraryFunctions.ml

```diff
@@ -130,7 +130,7 @@ let c_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("wcstombs", unknown ~attrs:[ThreadUnsafe] [drop "dst" [w]; drop "src" [r]; drop "size" []]);
     ("wcsrtombs", unknown ~attrs:[ThreadUnsafe] [drop "dst" [w]; drop "src" [r_deep; w]; drop "size" []; drop "ps" [r_deep; w_deep]]);
     ("mbstowcs", unknown [drop "dest" [w]; drop "src" [r]; drop "n" []]);
-    ("abs", unknown [drop "j" []]);
+    ("abs", special [__ "j" []] @@ fun j -> Math { fun_args = (Abs j) });
```

Comment:
Could we also support `labs` and `llabs`?

And what about the floating-point `fabs` (and `fabsf` and `fabsl`)?

Reply:
`fabs` (and `fabsf` and `fabsl`) are specified as math library functions already and are also handled by the float domain. There is just no refinement for their parameters yet. Support for `labs` and `llabs` still needs to be added.

Reply:
My bad `fabs` is indeed there. `labs` and `llabs` should just be matter of adding them here I suppose.

Reply:
Do we need to consider the different types for the evaluation as we do for the floating-point versions?

Reply:
I guess yes, because currently in `base` it uses `IInt` explicitly.

Reply:
There are now versions for `labs` and  `llabs`, with the corresponding `ikind`.